install-deps.sh

```diff
@@ -223,6 +222,8 @@ if [ x$(uname)x = xFreeBSDx ]; then
         www/node \
         www/npm \
         www/fcgi \
+        security/nss \
+	security/kbr5 \
```

Comment:
might want to spaces instead of tab for consistency (at least in this command).

Reply:
@tchaikov 
I'm going to rewrite this whole block anyways, since using such a long list on an system with already a lot of packages is a very slow process due to confilct comparison. So adding them 1 at the time is faster. So there's another PR comming for that.